src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -449,7 +449,7 @@ def _iter_rows(self) -> Iterable[Tuple[str, str, str, Union[float, int]]]:
         for side, rank_type in itt.product(SIDES, RANK_TYPES):
             for k, v in self.hits_at_k[side][rank_type].items():
                 yield side, rank_type, f"hits_at_{k}", v
-            yield side, rank_type, "rank_count", self.rank_count
+            yield side, rank_type, "rank_count", self.rank_count[side]
```

Comment:
With an attached tracker this line crashes the eval because `self.rank_count` is an empty dict (according to the debugger with a breakpoint here)

Reply:
I guess that is the same as this error: https://github.com/pykeen/pykeen/runs/4872388035?check_suite_focus=true#step:5:466